### ISSUE TITLE
[documentation] `series.json` schema has wrong field `ComicImage`

### ISSUE THREAD

**Issue by luevano** [mylar3/mylar3#1490]
**Describe the bug**
Documentation error.

in the [series.json schema](https://github.com/mylar3/mylar3/wiki/series.json-schema-%28version-1.0.1%29) it specifies that the comic image should be `comicImage` with lowercase "c", whereas the [examples](https://github.com/mylar3/mylar3/wiki/series.json-examples) show `ComicImage` with uppercase "C".

I report this as I was using the schema as a reference for an application I'm working on and I came across this while working with Komga integrations.

**CarePackage**
NA

**Environment (please complete the following information):**
NA

**Additional information**
NA

**Comment by evilhero** [comment]
Not sure why this is just being caught now - but you're totally correct.

We're actually creating the json as per the example (``ComicImage``), at some point the schema reference changed to lowercase (``comicImage``). 

Ideally I'd like to keep the casing as consistent as possible (snake_case), so I'll revision the schema so it's actually ``comic_image`` going forward for the [v1.0.2 schema.](https://github.com/mylar3/mylar3/wiki/series.json-schema-%28version-1.0.2%29)

**Comment by luevano** [comment]
> Not sure why this is just being caught now - but you're totally correct.
> 
> We're actually creating the json as per the example (`ComicImage`), at some point the schema reference changed to lowercase (`comicImage`).
> 
> Ideally I'd like to keep the casing as consistent as possible (snake_case), so I'll revision the schema so it's actually `comic_image` going forward for the [v1.0.2 schema.](https://github.com/mylar3/mylar3/wiki/series.json-schema-%28version-1.0.2%29)

Yeah I guess everyone looks at the examples first?, the software I forked had everything wrong and Komga didn't recognize it, at some point there was also `comicId` (uppercase "I").

I was going to suggest standardizing the schema but didn't want to push it with my issue. But now that you're on it, any plans on changing the following?

- `comicid` -> `comic_id` (both in the main `properties` and the `collects`)
- `issueid` ->  `issue_id`
- `booktype` -> `book_type`?

How does these changes to the schema start making it to the codebase? as in when should I expect it to start getting used? (so that I can start opening issues at Komga and making my changes).

**Comment by evilhero** [comment]
No plans on changing the ``comicid``, ``issueid`` and ``booktype`` fields as those are like that on various websites as single words and throughout the codebase as well (I mean whose to say if CV is right, or wrong really - this one time lol).

I'm getting ready to push the v1.0.2 schema generation to the codebase within the hr, so it will, by default be generating that schema version when it does generate the series.json. 

It will be in the python3-dev branch tho for abit prior to hitting master due to the influx of recent changes we did, but for those running dev (or nightly in lsio) the change will be available. If you're relying heavily on the json for data, you may want to capture the version of the schema and base your code off that to bypass any discrepencies in users' json files.

**Comment by luevano** [comment]
I think this was fixed, no? This could be closed.
